documentation/glossary.md

```diff
@@ -0,0 +1,56 @@
+# Glossary of aviation terms
+*Openscope uses some terms specific to aviation, this glossary attempts to define them.*
+---
+## ILS
+*abbr. of* Instrument Landing System
+
+A radio antenna at the end of the runway, known as a ''localizer'', guides pilots down onto the runway for landing, via an instrument known as a *guide scope*.
+
+## Fix
+
+A pre-defined point that describes an aircraft's current position. A fix can be an airport (`EGKK`), a waypoint (`WILLO`), or other points, such as a VOR station (`GWC`).
+
+## Waypoint
+
+A fix that is used to aid in navigation, defined in a flight plan or route. Waypoints are assigned a 5-letter code, such as `WILLO` and `ABTUM`.
+
+## Flight Level
+*abbr.* FL
+
+A plane's altitude, in 100s of feet. For example, a plane at `FL350` is flying at 35,000ft.
+
+## SID
+*abbr. of* Standard Instrument Departure
+
+A route that an aircraft will follow to exit the airport's radar control area. Each SID begins at the airport and terminates at a fix, with waypoints in between.
```

Comment:
Let's go for something more generic here to cover all bases.

> An instrument procedure used to guide aircraft from a departure airport out of that airport's airspace.

And then add a term _instrument procedure_:

> A charted series of instructions guiding aircraft along a specific path. This could be a series of headings to fly until passing certain altitudes, or a series of fixes to follow. Altitude and speed restrictions may often apply at certain points along the procedure as well. These are used by air traffic control to organize traffic on a large scale, and reduce the load on controllers under normal operations.